components/global/sdkInstallTab.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+	<div>
+		<div class="flex flex-row m-auto w-full mt-32px mb-24px border-b border-b-grey-10">
+			<li v-for="tab of tabs" :key="tab.id" class="mr-24px !list-none last-of-type:mr-0">
+				<button class="pb-10px flex items-center has-icon" :class="activeTab === tab.id ? 'active' : ''" @click="switchTabs(tab.id)">
+					<img :src="require(`~/assets/images/${tab.id}.svg`)" :alt="tab.label" class="h-16px w-16px mr-16px !my-0" />
+					<span class="text-14 text-left text-black tracking-[0.02em] mobile:min-w-[80px]">{{ tab.label }}</span>
+				</button>
+			</li>
+		</div>
+		<div>
+			<nuxt-content :document="pageData"></nuxt-content>
+		</div>
+	</div>
+</template>
+<script>
+export default {
+	layout: 'docs',
+	data() {
+		return {
+			pageData: '',
+			tabs: [
+				{ label: 'Javascript', id: 'javascript' },
+				{ label: 'Python', id: 'python' },
+				{ label: 'PHP', id: 'php' },
+				{ label: 'Ruby', id: 'ruby' },
+				{ label: 'Golang', id: 'golang' }
+			],
+			activeTab: 'javascript'
+		};
+	},
+	mounted() {
+		this.fetchPageData('install-js');
```

Comment:
@Youngestdev You should call `switchTabs()` here instead

Reply:
Or in addition. Can't find a relative 'install-js' site that you're calling here though

Reply:
Uhm. I have no idea how this component thing works or what `switchTabs()` does. 

The second comment: that’s a typo. I’ll fix that. 

Is that the cause of the issue with displaying multiple tabs? @horlah

Reply:
@Youngestdev  calling `this.switchTabs('javascript')` would be more efficient as it would retain the first tab as your active tab and also fetch the appropriate content on page load

Reply:
ok